zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2337,7 +2369,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/EvaluateDecisionRequest"
+              $ref: "#/components/schemas/EvaluateDecisionRequestBody"
```

Comment:
🔧 Same as above, the example needs to move to a string key as well.
❓ Same as above, @pepopowitz, do we need to adjust any automation in `camunda-docs` for this?